### PR TITLE
TW-665 Updates to multi-team

### PR DESCRIPTION
**Pull request by claireriley**
**Note**: You can ignore the Migration FAQs section after the first question, I just had to indent all of the answers to get the bullet list in the first question to indent correctly

### CODE REVIEW

**Review by petewitcher — APPROVED** [review]
LGTM!

**Review by HeatherStenson — APPROVED** [review]
Looks good! 🍊

**Review by petewitcher — APPROVED** [review]
LGTM!

File: src/pages/docs/administration/team-merge.md [hunk 1]
```diff
@@ -120,29 +120,42 @@ See [performing distributed migration](#performing-distributed-migration) to cho
...
+
+    Your documentation links will break since the earlier team is no longer active. New documentation URLs will need to be generated when collections are [republished from your new team pages](/docs/publishing-your-api/publishing-your-docs/). If you are [publishing to a custom domain](/docs/publishing-your-api/custom-doc-domains/), unpublish and remove the domain from your earlier team in order to add it to your new team and republish.
 
 * **What happens to the earlier team once everyone leaves?**
-When the last person leaves a team it's automatically deactivated. If there is only one Admin remaining on the team, they will automatically inherit all shared and unshared Postman data and can perform the bulk export.
 
-> Reach out to [Postman support](https://www.postman.com/support/) if you have any questions or run into any issues setting up Postman for your team.
+    When the last person leaves a team it's automatically deactivated. If there is only one Admin remaining on the team, they will automatically inherit all shared and unshared Postman data and can perform the bulk export.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/team-merge.md:156]: For free teams, If you're the last member to leave a team, the team will be deleted. You will no longer have access to the team's workspaces, including personal workspaces, or any elements in them. [Export your Postman data](https://github.com/postmanlabs/postman-docs/blob/TW-665-updates-to-multi-team/docs/getting-started/importing-and-exporting-data/#exporting-postman-data) prior to leaving the team to ensure it stays with you.

For paid teams, you need to contact support to cancel plan subscription, before you can leave the team.

**Review by ArjunSingh-PM — COMMENTED** [review]
The changes look good. Added a comment regarding the FAQ for last member leaving the team.

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Review by ArjunSingh-PM — APPROVED** [review]
The changes look good, Claire. Thanks!

**Pull request merged**